test.md

```diff
@@ -186,6 +186,27 @@ only need to do it on the first one. If you have a `<th>`, set it there.
   </tr>
 </table>
 
+## Box
+
+Here is the sample code for a single-cell table masquerading as a box. There are other ways to create a box (*e.g.*, with an HTML "box" element) but this works for our purposes.
+
+1. Lorem ipsum dolor sit amet, wisi viverra voluptas cras, vestibulum ligula molestie sagittis quis, nulla elementum id. Scelerisque tempus at consectetuer, neque nascetur erat pharetra rhoncus.
+2. Ut ridiculus congue pretium phasellus nulla arcu, litora quis. Facilisi habitasse a mollis dui accumsan egestas, massa ut vestibulum est sit ut, maecenas dolore rutrum vestibulum aenean fermentum, dolor posuere lectus, sit id vel erat penatibus wisi odio.
+3. Scelerisque nec fusce, orci risus, viverra vel nunc, non nulla vel integer praesent non, gravida in vestibulum. Turpis id ac, et dolor pede lectus volutpat nonummy elit, sed semper justo eget elementum, quisque non id duis lacus donec lorem, sit volutpat curabitur sed est.
+<table border="1px solid" style="background-color:#F5F5F5; border-color:#DCDCDC; margin-left:3%;">
```

Comment:
Is the table supposed to be inside the list item? If so, it needs to be indented to match the first line of text. It also needs a newline before it starts. Indentation is picky in Jenkins, hence the guidance above.